javacord-core/src/main/java/org/javacord/core/interaction/SlashCommandInteractionImpl.java

```diff
@@ -54,11 +57,21 @@ public SlashCommandInteractionImpl(DiscordApiImpl api, TextChannel channel, Json
                     }
                 });
             }
+
+            if (resolved.has("attachments")) {
+                resolved.get("attachments").fields().forEachRemaining(attachmentNode -> {
+                    JsonNode attachmentData = attachmentNode.getValue();
+                    Long id = Long.parseLong(attachmentNode.getKey());
+                    if (!resolvedAttachments.containsKey(id)) {
+                        resolvedAttachments.put(id, new AttachmentImpl(api, attachmentData));
+                    }
```

Comment:
```suggestion
                    resolvedAttachments.computeIfAbsent(Long.parseLong(attachmentNode.getKey()), id -> new AttachmentImpl(api, attachmentData));
```
With `computeIfAbsent` I think looks better

Reply:
I do agree. That section of the code is simply a copy of what was written for `resolvedUsers` as well.

Reply:
In that case it doesn't make sense to copy it from the users. The users need it because of how Javacord works (user == member object). If we did not do this check we would simply overwrite the user which contains the member.
It is not wrong, but also not needed. I would suggest to apply this suggestion for both (user / attachment) or just do a simple `put` for users as it looks cleaner.

Reply:
Actually yeah, that made sense. I think my brain went offline while looking at this suggestion. A simple `put` should just do the trick in this scenario.

Reply:
Applied the following changes, thanks for the input! 🧇